packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -430,7 +435,7 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
                 highlightIndexNotifier: _highlightedOptionIndex,
                 child: Builder(
                   builder: (BuildContext context) {
-                    return widget.optionsViewBuilder(context, _select, _options);
+                    return widget.optionsViewBuilder(context, _select, _options, _fieldWidth);
```

Comment:
Another option that crossed my mind is to wrap this in a SizedBox or ConstrainedBox instead of passing in the _fieldWidth.  However, I think some users of Autocomplete won't want their options to match the width of the field, they'll want them to be wider.  Because of that, you're probably right that we should pass the width and give them the option to do what they want :+1: